missing pair_uint64_bytes.first (type=uint64)

I'm getting another failing unit test with the latest CDT version for vRAM.

```
  15 passing (2m)
  1 failing

  1) IPFS Service Test Contract
       IPFS Load Manifest:
     Error: missing pair_uint64_bytes.first (type=uint64)
      at Object.serializeStruct [as serialize] (node_modules/eosjs/dist/eosjs-serialize.js:571:27)
      at Object.serializeArray [as serialize] (node_modules/eosjs/dist/eosjs-serialize.js:640:26)
      at Object.serializeStruct [as serialize] (node_modules/eosjs/dist/eosjs-serialize.js:564:28)
      at Object.serializeStruct [as serialize] (node_modules/eosjs/dist/eosjs-serialize.js:564:28)
      at serializeActionData (node_modules/eosjs/dist/eosjs-serialize.js:1041:12)
      at Object.serializeAction (node_modules/eosjs/dist/eosjs-serialize.js:1051:15)
      at Api.<anonymous> (node_modules/eosjs/dist/eosjs-api.js:278:71)
      at step (node_modules/eosjs/dist/eosjs-api.js:47:23)
      at Object.next (node_modules/eosjs/dist/eosjs-api.js:28:53)
      at fulfilled (node_modules/eosjs/dist/eosjs-api.js:19:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

### Steps to reproduce:

Install zeus and unbox ipfs-dapp-service box.

```
npm i -g @liquidapps/zeus-cmd
mkdir ipfs-dapp-service
cd ipfs-dapp-service
zeus box create
zeus unbox ipfs-dapp-service
# update CMake files in ./contracts/eos as needed to support new cdt
zeus compile # or cmake . && make in ./contracts/eos
zeus test ipfsconsumer --services ipfs --chain eos --single-chain
```

See https://github.com/AntelopeIO/cdt/issues/15 for additional install instructions.

Last version this worked on was 3.0.0. Did not work on 3.0.1 with same error.

I have built and installed newest CDT from main.
Currently I have following error when I try to build the contracts:
```
$ make
[  1%] Performing build step for 'dappservices'
[ 50%] Building CXX object CMakeFiles/dappservices.dir/dappservices.cpp.obj
In file included from /home/mikel/tmp/ipfs-dapp-service/contracts/eos/dappservices/dappservices.cpp:1:
In file included from /home/mikel/tmp/ipfs-dapp-service/contracts/eos/dappservices/./dappservices.hpp:8:
In file included from /usr/opt/cdt/3.2.0-dev/bin/../include/eosiolib/contracts/eosio/eosio.hpp:8:
/usr/opt/cdt/3.2.0-dev/bin/../include/eosiolib/contracts/eosio/multi_index.hpp:780:33: error: use of undeclared identifier 'get_type'
         using type = decltype( get_type(std::make_index_sequence<Num>{}) );
                                ^
```
although get_type is declared in line 772 (line above).
Maybe newer standard has to be used?

--- a/docs/08_troubleshooting/index.md
+++ b/docs/08_troubleshooting/index.md
@@ -168,7 +168,7 @@ The second parameter is the system's symbol (i.e. for main net this is `EOS`). I
 
 ## Backward incompatible change in generating abi for maps
 
-In CDT version 3.0.1, there has been a backward incompatible change in generating ABI for std::map. The change is as follows:
+In CDT version 3.0.1, there has been breaking ABI change in std::map. The change is as follows:
 
 Old Version (CDT 3.0.0 or earlier):
 ```

--- a/docs/08_troubleshooting/index.md
+++ b/docs/08_troubleshooting/index.md
@@ -165,3 +165,39 @@ assertion failure with message: system contract must first be initialized
 
 The failure is stating that `eosio.system` `init` action was not called yet. The `init` action is implemented by the `void init(uint64_t, symbol)` function. The first parameter is the version, this should always be `0` for now, until a new version of `init` will be created that handles more information.
 The second parameter is the system's symbol (i.e. for main net this is `EOS`). If you followed the [BIOS Boot Sequence](https://docs.eosnetwork.com/docs/latest/tutorials/bios-boot-sequence) tutorial and created a system with the default symbol `SYS` then `SYS` shall be used as the system's symbol in the `init` action. It is whatever symbol you as the chain creator want to use in your `Antelope` based blockchain.
+
+## Backward incompatible change in generating abi for maps
+
+In CDT version 3.0.1, there has been a backward incompatible change in generating ABI for std::map. The change is as follows:
+
+Old Version (CDT 3.0.0 or earlier):
+```
+  "fields": [
+      {
+          "name": "key",
+          "type": "<map first type>"
+      },
+      {
+          "name": "value",
+          "type": "<map second type>"
+      }
+  ]
+```
+
+New Version (CDT 3.0.1 and above):
+```
+  "fields": [
+      {
+          "name": "first",
+          "type": "<map first type>"
+      },
+      {
+          "name": "second",
+          "type": "<map second type>"
+      }
+  ]
+```
+
+As a result, if you are using code based on CDT 3.0.0 or earlier, you may encounter errors when upgrading to CDT 3.0.1 and above.
+An example error message could be: `Error: missing pair_uint64_bytes.first (type=uint64)`.
+To resolve this issue, you need to update your code to use `first` and `second` to access the elements of `std::map`, instead of the old `key` and `value` identifiers.

